### PR TITLE
Feature/wen 31

### PR DESCRIPTION
**Pull request by AivarasRinkevicius**
Created slider, global header and adjusted styling.

### CODE REVIEW

**Comment by ernestas20111** [comment]
Remove .idea files, I think they should be added to gitignore

File: flower-e-shop-frontend/src/app/components/product-description/product-description.component.html [hunk 1]
```diff
@@ -31,5 +31,5 @@ <h1>{{product.productName}}</h1>
 </ng-template>
 
 <ng-template #doesntExist>
-  <p>Ieškoma prekė neegzistuoja!</p>
+  <body>Ieškoma prekė neegzistuoja!</body>
```
Comment (ernestas20111) [inline flower-e-shop-frontend/src/app/components/product-description/product-description.component.html:34]: Why this part was changed from p to body?

File: flower-e-shop-frontend/src/app/app.component.html [hunk 2]
```diff
@@ -1 +1,3 @@
-<router-outlet></router-outlet>
+<router-outlet>
+  <app-header></app-header>
```
Comment (ernestas20111) [inline flower-e-shop-frontend/src/app/app.component.html:2]: Why it is inside router outlet? Shouldn't it be outisde of it and below or above it?

**Review by ernestas20111 — COMMENTED** [review]
fgd

File: flower-e-shop-frontend/src/app/app.module.ts [hunk 3]
```diff
@@ -11,17 +11,21 @@ import { ProductComponent } from './components/product/product.component';
 import { SearchFilterPipe } from './pipes/search-filter/search-filter.pipe';
 import { SearchFieldComponent } from './components/search-field/search-field.component';
 import { MainPageComponent } from './components/main-page/main-page.component';
-import {ProductDescriptionComponent} from "./components/product-description/product-description.component";
+import {ProductDescriptionComponent} from './components/product-description/product-description.component';
```
Comment (viliusmac) [inline flower-e-shop-frontend/src/app/app.module.ts:14]: looks good

**Comment by viliusmac** [comment]
> I like big black men

looks good

File: flower-e-shop-frontend/src/app/components/product-description/product-description.component.html (hunk 1 above)
Comment (AivarasRinkevicius) [reply flower-e-shop-frontend/src/app/components/product-description/product-description.component.html:34]: Lol

File: flower-e-shop-frontend/src/app/app.component.html (hunk 2 above)
Comment (AivarasRinkevicius) [reply flower-e-shop-frontend/src/app/app.component.html:2]: Yep, good catch

**Comment by AivarasRinkevicius** [comment]
>

**Pull request merged**